packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -1874,27 +1874,27 @@ describe("BnsConnection", () => {
 
     // create the preimages for the three swaps
     const preimage1 = Encoding.toAscii("the first swap is easy");
-    // const hash1 = new Sha256(preimage1).digest();
+    const hash1 = new Sha256(preimage1).digest();
```

Comment:
Yeah, I really don't want to have to leak this all over the place, especially if we start putting checks and validation in there... see above comment about preimageIdentifier